react-native-libraries.json

```
@@ -9965,5 +9965,13 @@
     "examples": ["https://github.com/ekreative/react-native-braintree/tree/master/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/okwasniewski/react-native-menubar-extra",
+    "npmPkg": "react-native-menubar-extra",
+    "examples": ["https://github.com/okwasniewski/react-native-menubar-extra/tree/main/example"],
+    "images": ["https://raw.githubusercontent.com/okwasniewski/react-native-menubar-extra/main/img/example-usage.png"],
+    "macos": true,
+    "newArchitecture": true
```

Hey @okwasniewski, thanks for adding new macOS entry!

```suggestion
  {
    "githubUrl": "https://github.com/okwasniewski/react-native-menubar-extra",
    "examples": ["https://github.com/okwasniewski/react-native-menubar-extra/tree/main/example"],
    "images": ["https://raw.githubusercontent.com/okwasniewski/react-native-menubar-extra/main/img/example-usage.png"],
    "macos": true
```

Two nits, `npmPkg` can be skipped since it matches GH repo name, and `newArchitecture` will be autoassigned, becouse your `package.json` contains codegen entry.

Let's update that, and we can ship. 🙂

---

Applied your suggestion! Thanks for review